### PR TITLE
aws provider v3.0 support

### LINKED ISSUE #20: Support AWS provider v3
## Describe the Bug
AWS provider v.3 is not supported

```
No provider "aws" plugins meet the constraint " < 4.0,>= 2.23,~> 2.0,~> 3.0".

The version constraint is derived from the "version" argument within the
provider "aws" block in configuration. Child modules may also apply
provider version constraints. To view the provider versions requested by each
module in the current configuration, run "terraform providers".

To proceed, the version constraints for this provider must be relaxed by
either adjusting or removing the "version" argument in the provider blocks
throughout the configuration.


Error: no suitable version is available
```

```
terraform providers
.
├── provider.aws ~> 3.0
└── module.ftp_bucket
    ├── provider.aws (inherited)
    ├── provider.random
    ├── module.aggregated_s3_bucket_policy
    │   └── provider.aws ~> 2.0
```

Seems provider version constraint should be updated to something like
```
  required_providers {
    aws = ">= 2.23, < 4.0"
  }
}
```

### PR DESCRIPTION
**Pull request by savealive**
changed aws provider constraint to support v3.0

## what
Changed versions.tf to supprort aws provider v3.0

## references
closes #20

### CODE REVIEW

File: versions.tf [hunk 1]
```diff
@@ -2,6 +2,6 @@ terraform {
   required_version = ">= 0.12.0, < 0.14.0"
```
Comment (aknysh) [inline versions.tf:2]:
```suggestion
  required_version = ">= 0.12.0"
```

File: versions.tf [hunk 2]
```diff
@@ -2,6 +2,6 @@ terraform {
...
 
   required_providers {
-    aws = "~> 2.0"
+    aws = ">= 2.23, < 4.0"
```
Comment (aknysh) [inline versions.tf:5]:
```suggestion
    aws = ">= 2.23"
```

**Review by aknysh — COMMENTED** [review]
the latest decision was not to add the ceilings since it's difficult to update and test all the modules

**Comment by woz5999** [comment]
@aknysh i've made the requested changes in my PR #22

**Review by aknysh — APPROVED** [review]
thanks @savealive

**Pull request merged**